crates/neon-runtime/native/src/neon.cc

```diff
@@ -8,6 +8,7 @@
 #include "neon_string.h"
 #include "neon_class_metadata.h"
 #include "neon_task.h"
+#include "neon_threadsafe_cb.h"
```

Comment:
thread safe callback is used for the internal API as it's not event handler specific